templates/etc/cron.hourly/create_home_directories.rb.erb

```diff
@@ -36,7 +36,7 @@ File.umask(0122)
 
 ldap_params = Hash.new
 
-servers = (["<%= Array(@uri).map { |x| x = x.split("://").last }.join(',') %>"])
+servers = (<%= Array(@uri).map { |x| x = x.split("://").last } %>)
```

Comment:
This works, but it's taking advantage of the fact that Ruby 1.9+ prints the brackets around Arrays when converting to a string. We need to make sure this is literally enforced.